src/pages/landing/projects/components/NoteCard.tsx

```diff
@@ -0,0 +1,110 @@
+import { useState } from 'react';
+import { Text, Box, Flex, Link, useBreakpointValue } from '@chakra-ui/react';
+
+const ImageWithReload = ({ src, alt, ...rest }) => {
+  const [error, setError] = useState(false);
+
+  const handleError = () => {
+    setError(true);
+  };
+
+  return (
+    <Box
+      position="relative"
+      overflow="hidden"
+      width="100%"
+      height="100%"
+      backgroundColor="neutral.200"
+    >
+      <img
+        src={error ? '' : src}
+        alt={alt}
+        onError={handleError}
+        style={{
+          width: '100%',
+          height: '100%',
+          objectFit: 'contain',
+        }}
+        {...rest}
+      />
+    </Box>
+  );
+};
+
+interface Props {
+  content: string;
+  nostrich: string;
+  date: string;
+}
+
+export default function NoteCard({ content: rawContent, nostrich, date }: Props) {

```

Comment:
Change name to NostrProjectCard, and we use ES6 style arrow functions, let's stick to that standard.